client/my-sites/stats/pages/subscribers/index.tsx

```diff
@@ -99,6 +102,18 @@ const StatsSubscribersPage = ( { period }: StatsSubscribersPageProps ) => {
 		[]
 	); // Track the last viewed tab.
 
+	useEffect( () => {
+		const query = context.query;
+		recordCurrentScreen(
+			'subscribers',
+			{
+				queryParams: query,
+				period: period?.period || null,
```

Comment:
The `period?.period` seems [not to need](https://github.com/Automattic/wp-calypso/blob/93552eecea071dfdea200b13944e2e49df98573d/client/my-sites/stats/pages/subscribers/controller.tsx#L27) the optional operator.
```suggestion
				period: period.period,
```